pkgs/games/xjump/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, gcc, autoconf, automake, xorg, ... }:
```

Comment:
I don't know that it's problematic, but the "..." is certainly not idiomatic unless needed or extra arguments expected.

Please remove or explain :).

Reply:
Also, `callPackage` automatically searches `xorg`, so it might be better to have `libX11`, `libXt`, `libXpm` and `libXaw` as inputs.